locales/de/translation.json

```
@@ -114,16 +114,16 @@
  "Give support on the <a href='https://forum.luanti.org'>forums</a> and <a\n\t\t\t\t\t\t\t\thref='https://docs.luanti.org/about/irc/'>IRC</a>.": "Bieten Sie Unterstützung in den <a href='https://forum.luanti.org'>Foren</a> und im <a\n\t\t\t\t\t\t\t\thref='https://docs.luanti.org/about/irc/'>IRC</a>.",
  "Help translate Luanti using\n\t\t\t\t\t\t\t<a href='https://hosted.weblate.org/projects/minetest/minetest/'>our web interface</a>.": "Helfen Sie mit, Luanti zu übersetzen, indem Sie\n\t\t\t\t\t\t\t<a href='https://hosted.weblate.org/projects/minetest/minetest/'>unsere Webschnittstelle</a> nutzen.",
  "You can find help with any problems [from our community](/en/get-involved/).\n": "Bei Problemen finden Sie Hilfe [in unserer Community](/en/get-involved/).",
- "<a href='https://forum.luanti.org/viewtopic.php?f=7&t=19877'>Help us review and merge\n\t\t\t\t\t\t\t\tpull requests</a>.": "<a href='https://forum.luanti.org/viewtopic.php?f=7&t=19877'>Helfen Sie uns bei der Überprüfung und Zusammenführung\n\t\t\t\t\t\t\t\tpull requests</a>.",
- "<a href='#donate'>Donate</a> money to pay for infrastructure, and\n\t\t\t\t\t\t\t<a href='https://content.luanti.org/donate/'>support your favourite contributors</a>.": "<a href='#donate'>Spenden Sie</a> Geld für die Infrastruktur und\n\t\t\t\t\t\t\t<a href='https://content.luanti.org/donate/'>unterstützen Sie Ihre Lieblingsmitwirkenden</a>.",
+ "<a href='https://forum.luanti.org/viewtopic.php?f=7&t=19877'>Help us review and merge\n\t\t\t\t\t\t\t\tpull requests</a>.": "<a href='https://forum.luanti.org/viewtopic.php?f=7&t=19877'>Helfen Sie uns bei der Überprüfung und Zusammenführung\n\t\t\t\t\t\t\t\tvon pull requests</a>.",
```

(I'm also not sure whether "Zusammenführung" is a good translation for merging pull requests but 🤷)

```suggestion
 "<a href='https://forum.luanti.org/viewtopic.php?f=7&t=19877'>Help us review and merge\n\t\t\t\t\t\t\t\tpull requests</a>.": "<a href='https://forum.luanti.org/viewtopic.php?f=7&t=19877'>Helfen Sie uns bei der Überprüfung und Zusammenführung\n\t\t\t\t\t\t\t\tvon Pull Requests</a>.",
```

---

I did some research and found that GitHub also uses "Zusammenführung" as a translation. See here: https://docs.github.com/de/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request